build.gradle

```diff
@@ -209,6 +209,14 @@ tasks.register("branchConsistency") {
   dependsOn ":verifyVersions", ":verifyBwcTestsEnabled"
 }
 
+tasks.register("dependenciesGraph") {
```

Comment:
This is just creating a "dummy" task on the root build proejct. This logic should in the `DependenciesGraphPlugin` or `DependenciesGraphTask` itself.

Reply:
Yeah I guess I misunderstood you and I'm still not clear what your suggestion is. Is startParameter.isOffline available somehow for me to check while in the plugin or in the Task class ?

Reply:
Nevermind, I think I got it